test/kwtool_web/views/api/v1/error_view_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule KwtoolWeb.Api.V1.ErrorViewTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
+
+  alias KwtoolWeb.Api.V1.ErrorView
+
+  defmodule Device do
+    use Ecto.Schema
+
+    import Ecto.Changeset
+
+    schema "devices" do
+      field :device_id, :string
+      field :operating_system, :string
+      field :device_name, :string
+
+      timestamps()
+    end
+
+    def changeset(device \\ %__MODULE__{}, attrs) do
+      device
+      |> cast(attrs, [
+        :device_id,
+        :operating_system,
+        :device_name
+      ])
+      |> validate_required([
+        :device_id,
+        :operating_system,
+        :device_name
+      ])
+    end
+  end
+
+  test "renders 404.json" do
```

Comment:
Let's put all the test cases here inside `describe` block

Reply:
Wrapped in https://github.com/byhbt/kwtool/pull/45/commits/9c44d18b14f6ef16749063289d5ec0cab7b29835